controllers/show.go

```diff
@@ -47,13 +49,54 @@ func (sc *ShowController) GetShow(w http.ResponseWriter, r *http.Request) {
 
 	// Needed so that credits are grouped by type
 
+	var scheduledSeasons = make([]myradio.Season, 0)
+	var timeslots = make([]myradio.Timeslot, 0)
```

Comment:
could set the capacity of these to the length of seasons, the vast majority are going to be the same length (alternatively, just remove any unscheduled from the inital array)